region/maridia/outer.json

```diff
@@ -5134,6 +5293,26 @@
                   "name": "Base",
                   "notable": false,
                   "requires": [ "Morph" ]
+                },
+                {
+                  "name": "G-Mode Morph",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "SpringBall",
+                      "Bombs",
+                      {"and": [
+                        "canBombHorizontally",
+                        {"ammo": { "type": "PowerBomb", "count": 3 }}
+                      ]}
+                    ]}
+                  ],
+                  "note": "Use spring ball to cross the room, or use bombs to kill the Zebbos so they stop respawning. One PB can be used below the corner in the ceiling to bomb across the farm while killing the Zebbos."
```

Comment:
I like bomb jumping from the bottom leftmost plant to jump over the zebbos.  Starting from under the corner gets me hit most of the time.
You could use an extra pb instead of bomb horizontally too.
Killing them with bombs is very not safe.  But bouncing on the bombs to avoid the zebbos helps.